kai_mcp_solution_server/tests/test_multiple_integration.py

```
@@ -353,3 +359,183 @@
+                loop = asyncio.new_event_loop()
+                asyncio.set_event_loop(loop)
+
+            try:
+                result = loop.run_until_complete(fn(*args, **kwargs))
+                return result
+            finally:
+                loop.close()
+
+        async def client_task(client_id: str) -> None:
+            print(f"[Client {client_id}] starting")
+            ssl_patch = apply_ssl_bypass()
```

Ruff (0.12.2)

Local variable `ssl_patch` is assigned to but never used

Remove assignment to unused variable `ssl_patch`

(F841)